### PR TITLE
Git ssh

### PR DESCRIPTION
**Pull request by creeonix**

### CODE REVIEW

File: packages.js [hunk 1]
```diff
@@ -44,48 +51,34 @@ Packages.fromFile = function (file, callback) {
   });
 };
 
-// Create a packages document per tarball url.
-// { tarUrl: { packageName: tarPath, .. }, ... }
-var getTarballDict = function (packages) {
-  var tarballs = {};
+// Create a packages document per git url.
+// { gitRepo: { vesion: { packageName: packagePath, .. }, ... } }
```
Comment (jperl) [inline packages.js]: Slight mispelling `vesion` -> `version`

**Comment by jperl** [comment]
Overall this looks really good. Does this work if you set version to a commit hash?

**Comment by creeonix** [comment]
yes, see tests

**Comment by jperl** [comment]
Ah excellent. K I'm going to merge this and get travis to work. Thanks!

**Comment by jperl** [comment]
Tests are now passing. Bumped `mgp` to `v1.0.0` -- thanks @creeonix!

**Pull request merged**